### DISCUSSION TITLE
Slider values get reset after changing another one

### DISCUSSION THREAD

**Discussion by lborcard** [marimo-team/marimo#1983] [Q&A]
### Describe the bug

I have 3 sliders and when I change one of them the other two get reset to base values but not the other way around. How can I prevent that from happening?

### Environment

{
  "marimo": "0.6.15",
  "OS": "Linux",
  "OS Version": "4.18.0-553.8.1.el8_10.x86_64",
  "Processor": "x86_64",
  "Python Version": "3.11.9",
  "Binaries": {
    "Browser": "--",
    "Node": "--"
  },
  "Requirements": {
    "click": "8.1.7",
    "importlib-resources": "missing",
    "jedi": "0.19.1",
    "markdown": "3.6",
    "pymdown-extensions": "10.8.1",
    "pygments": "2.18.0",
    "tomlkit": "0.12.5",
    "uvicorn": "0.30.1",
    "starlette": "0.37.2",
    "websockets": "12.0",
    "typing-extensions": "4.12.1",
    "black": "23.12.1"
  }
}

**Comment by akshayka** [comment]
I've moved this to a discussion because it doesn't sound like a bug. Probably the two sliders that are being reset are declared in a cell that uses the third slider. If you haven't already, I suggest opening `marimo tutorial intro` and `marimo tutorial ui` at the command line. If you want further help, please post source code and a video demonstrating what is happening, and a clear description of the behavior that you'd instead prefer.

**Reply by lborcard** [reply]
Thank you for your answer and I should have used another tag for my question as I figured it was part of the design but it still did not make sense because only one depends on the other and both get reset. Can this be because the whole cell is being fired not just the ui element?

**Reply by akshayka** [reply]
Yes, that is likely it
